.github/workflows/publish.yaml

```diff
@@ -23,17 +25,20 @@ jobs:
           cache-dependency-path: 'package-lock.json'
       - run: npm ci
       - name: Update version in package.json
+        env:
+          NEW_VERSION: ${{ github.event.release.tag_name }}
         run: |
-          NEW_VERSION=${{ github.event.release.tag_name }}
-          npm version $NEW_VERSION --no-git-tag-version
+          npm version "${NEW_VERSION}" --no-git-tag-version
       - name: Update generated examples
         run: npm run update-examples
       - name: Commit updated package.json
+        env:
+          VERSION_TAG: ${{ github.event.release.tag_name }}
         run: |
           git config --global user.name 'github-actions[bot]'
           git config --global user.email 'github-actions[bot]@users.noreply.github.com'
           git add package.json package-lock.json examples/
-          git commit -m "Update package.json version to ${{ github.event.release.tag_name }}"
+          git commit -m "Update package.json version to ${VERSION_TAG}"
```

Comment:
In other scripts, I've used `$VERSION_TAG` without braces. Does this have the same result?